service/routes/test.js

```diff
@@ -77,6 +83,10 @@ function createEndpoint(type, polyfillio) {
 	};
 }
 
-module.exports = {
-	createEndpoint: createEndpoint
-};
+router.use('/libs/mocha', express.static(path.join(__dirname, '/../../node_modules/mocha')));
```

Comment:
I think `require.resolve('mocha')` does the same job as `path.join(__dirname, '/../../node_modules/mocha')`